recipe/meta.yaml

```diff
@@ -0,0 +1,132 @@
+{% set name = "evalml" %}
+{% set version = "0.82.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f1fdcd55591db2c71f36fd19564b4716d29dfa438bb5185f08af679d3a4147e3
+
+build:
+  # skip these because of vowpalwabbit not supporting these
+  skip: True  # [win or s390x or py<38]
+  skip: True  # [osx and x86_64 and py==38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - numpy >=1.22.0
+    - pandas >=1.5.0,<2.1.0
+    - scipy >=1.5.0
+    - scikit-learn >=1.3.0
+    - scikit-optimize >=0.9.0
+    - pyzmq >=20.0.0
+    - colorama >=0.4.4
+    - cloudpickle >=1.5.0
+    - click >=8.0.0
+    - shap >=0.42.0
+    - statsmodels >=0.12.2
+    - texttable >=1.6.2
+    - woodwork >=0.22.0
+    - dask-core >=2022.2.0,!=2022.10.1
+    - distributed >=2022.2.0,!=2022.10.1
+    - featuretools >=1.16.0
+    - nlp_primitives >=2.9.0
+    - networkx >=2.6,<3.2
+    - plotly >=5.0.0
+    - python-kaleido >=0.1.0
+    - ipywidgets >=7.5
+    - xgboost >=1.7.0
+    - catboost >=1.1.1
+    - lightgbm >=4.0.0
+    - matplotlib-base >=3.3.3
+    - python-graphviz >=0.13  # [not win]
+    - seaborn >=0.11.1
+    - category_encoders >=2.2.2,<=2.5.1.post0
+    - imbalanced-learn >=0.11.0
+    - pmdarima >=1.8.5
+    - sktime >=0.21.0
+    - lime >=0.2.0.1
+    - vowpalwabbit >=8.11.0
+    - tomli >=2.0.1
+    - packaging >=23.0
+    - black >=22.3.0
+    - holidays >=0.13,<0.21
+    - jupyter
+
+# Skip some tests:
+# AssertionError: assert '$PREFIX/lib/python3.9/site-packages/evalml' == '$SRC_DIR/evalml'
+{% set tests_to_skip = "test_get_evalml_root" %}
```

Comment:
I think all this jinja should go in the test section. Fair enough to skip what's being skipped, since it's a python-only package and if they're not passing it's not our fault. However I think this could all be made a lot neater.

It may also be worth communicating these test failures to Snowflake, I guess @M-Waszkiewicz-Anaconda or Sheetal would be best to judge about this though.

Reply:
You could also consider just not running the upstream tests

Reply:
Although since you've done this work now, it's probably nice to keep this info in as a ledger of what's failing. Perhaps report it upstream too.

Reply:
>You could also consider just not running the upstream tests

Tests are not being run. I've let this here to serve as some kind of documentation.